### PR TITLE
New Medium Salvage Map- Cultist Vault

### PR DESCRIPTION
**Pull request by 5150Con**
## About the PR 
This is my attempt at contributing a new salvage map to the community, I wanted to add some more themed salvages to the list, and this it it.

The salvage contains;

Persistent items:
1x Uranium Gen
1x Substation
1x Space drugs
1x Devil's Kiss drink
1x RGB plushie
1x Nar 'Sie plushie
Various floor cables
Various Cultist clothing

Spawners:
2x Salvage Mob 
1x Space Tick 
1x Space Bear 
1x Alien artifact
2x Salvage Material crate

I think this would be great because there isn't a similar salvage like it, and it has some power gen options for Salvage/Engi/Cargo. I tried to make it so that Salvage will need to construct a button/lever to open the vault, so it should involve a little more then loot and leave.

**Screenshots**

:cl:
- add: New Medium Salvage Map

### CODE REVIEW

File: Content.MapRenderer/Program.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ namespace Content.MapRenderer
 {
     internal class Program
     {
-        private const string MapsAddedEnvKey = "FILES_ADDED";
+        private const string MapsAddedEnvKey = MyStation.yml ;
```
Comment (mirrorcult) [inline Content.MapRenderer/Program.cs:15]: revert this change

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
i believe the actual size here is 6.5? not 10.5

**Comment by mirrorcult** [comment]
or its 8, actually, cant count

**Comment by mirrorcult** [comment]
actual salvage is good btw

File: Content.MapRenderer/Program.cs (hunk 1 above)
Comment (5150Con) [reply Content.MapRenderer/Program.cs]: I  have done so.

**Comment by 5150Con** [comment]
> or its 8, actually, cant count

I switched it to 8, I am not 100% sure on how to measure, as I just did furthest tile in one direction and put that

File: Resources/Prototypes/Maps/salvage.yml [hunk 2]
```diff
@@ -85,6 +85,12 @@
   mapPath: /Maps/Salvage/medium-pirate.yml
   size: 7.5
 
+- type: salvageMap
+  id: mediumCultVault
+  name: "Cult Vault"
+  mapPath: /Maps/Salvage/medium-cult-vault.yml
```
Comment (mirrorcult) [inline Resources/Prototypes/Maps/salvage.yml:91]: this isnt the correct path? you have it as `Med-Cult-Vault.yml`

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Pull request closed without merging**